Adding changelog for RequireJS fix (#1354)
twilio/twilio-video.js: CHANGELOG.md

    import * as Video from 'twilio-video';
  ```

Bug Fixes
---------

> - Fixed an issue where `Video` object is not exported properly when using RequireJS. (JSDK-3129)

Reviewer: Nitpick:
```suggestion
- Fixed a bug where the `Video` namespace is not exported properly when using RequireJS. (JSDK-3129)
```
